tasks/mysql.yml

```diff
@@ -23,6 +23,9 @@
 
 - name: "MySQL | Create database user"
   mysql_user:
+    login_host: "{{ zabbix_server_mysql_login_host | default(omit) }}"
+    login_user: "{{ zabbix_server_mysql_login_user | default(omit) }}"
```

Comment:
For completeness I would probably add `login_port` (and potentially `login_unix_socket` too)